python/tk_framework_alias_utils/plugin_bootstrap.py

```diff
@@ -109,6 +140,21 @@ def toolkit_plugin_bootstrap(
     # Log message to Alias prompt indicating that Flow Production Tracking is ready
     engine.alias_py.log_to_prompt("Flow Production Tracking initialized")
 
+    # Clean up any spawned processes before starting the Qt app
+    if powershell_parent_process:
+        try:
+            powershell_parent_process.kill()
```

Comment:
I see a lot of "return" between creating the process and this block.

Is it possible to have this in a context manager (with) instead to make sure the process is killed no matter what?

Reply:
Hm, I only see one return on line `109`, am I missing more?

Reply:
Instead of a context manager, you can have a try/finally block so it won't change your code too much